src/KubernetesClient/FileUtils.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.IO.Abstractions;
+
+namespace k8s
+{
+    public static class FileUtils
```

Comment:
```suggestion
    internal static class FileUtils
```

Reply:
or I think make `IFileSystem` a field of config object which would be better than global static.
but for test purpose, it is good.